src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -501,56 +383,18 @@ std::string statisticsTreat
   char  listBuffer[1024];
   bool  reqSemTaken;
 
+  // FIXME P10: should this be using the cacheSemTake()/cacheSemGive() functions ?
```

Comment:
Before, the sub.cache used the REQ sem.
This implementation might be from then and we should use the cache semaphore now.